src/lib/account_update.ts

```diff
@@ -780,7 +781,7 @@ class AccountUpdate implements Types.AccountUpdate {
     layout: AccountUpdatesLayout = AccountUpdate.Layout.NoDelegation
   ) {
     makeChildAccountUpdate(this, childUpdate);
-    this.isDelegateCall = Bool(false);
+    this.body.callType = AccountUpdate.CallType.ChildrenMayUseMyToken;
```

Comment:
API for controlling call type will be iterated on in the next PR (which adds `Blind_call`)

Reply:
Update: went with existing names for now, since it seems the `ChildrenMayUseMyToken` semantics weren't intended